src/Sylius/Component/Core/OrderProcessing/OrderShipmentProcessor.php

```diff
@@ -75,7 +76,7 @@ public function process(BaseOrderInterface $order)
     }
 
     /**
-     * @param BaseOrderInterface $order
+     * @param BaseOrderInterface | OrderInterface $order
```

Comment:
👍  ok, but it should definitely be `BaseOrderInterface|OrderInterface $order`, without spaces around `|` 😄